src/Views/User/_User.scss

```
@@ -1,10 +1,71 @@
 .userContainer {
   display: grid;
   grid-template-columns: 1fr 1fr;
-  grid-template-rows: auto auto;
-}
+  align-items: flex-start;
+
+  .usersBooks {
+    width: 350px;
+    margin: 25px;
+    justify-self: end;
+  }
+
+  .userPanel {
+    justify-self: flex-start;
+    margin: 25px;
+  }
+
+  .userInfo {
+    width: 350px;
+    height: 150px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    text-align: center;
+    background: $primary-gradient;
+    @include boxshadow;
+    padding: 10px;
+    margin-bottom: 25px;
+
+    button {
+      width: auto;
+    }
 
-.usersBooks {
-  grid-row: span 2;
+    .or::before,
+    .or::after {
+      display: inline-block;
+      content: "";
+      border-top: 1.5px solid black;
+      width: 100px;
+      margin: 0 1rem;
+      transform: translateY(-0.5rem);
+    }
```

it looks bad on width < 320px
Try this. It should look fine - commented line can be delete

```suggestion
    .or {
      display: flex;
      width: 100%;
     justify-content: space-between; // or space-evenly
      align-items: baseline;
    }

    .or::before,
    .or::after {
      // display: inline-block;
      content: "";
      border-top: 1.5px solid black;
      width: 40%;
      // margin: 0 1rem;
      transform: translateY(-0.5rem);
    }
```